vllm/transformers_utils/config.py

```diff
@@ -50,10 +51,12 @@ def get_config(model: str,
         config = config_class.from_pretrained(model,
                                               revision=revision,
                                               code_revision=code_revision)
-    if rope_scaling is not None:
-        logger.info("Updating rope_scaling from %r to %r",
-                    getattr(config, "rope_scaling", None), rope_scaling)
-        config.update({"rope_scaling": rope_scaling})
+    for key in ["rope_scaling", "rope_theta"]:
+        if locals()[key] is not None:
+            logger.info("Updating %s from %r to %r", key,
+                        getattr(config, key, None),
+                        locals()[key])
+            config.update({key: locals()[key]})
```

Comment:
I would prefer not to use `locals()` as it hurts readability. I think you can change it to:

```
for name, value in [("rope_scaling", rope_scaling), ("rope_theta", rope_theta)]:
    if value is not None:
        logger.info....
```